includes/Modules/Analytics_4.php

```diff
@@ -2390,4 +2410,30 @@ private function has_audience_site_kit_identifier( $data, $identifier ) {
 
 		return false;
 	}
+
+	/**
+	 * Gets the list of site kit created audiences.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param array $audiences List of audiences.
+	 *
+	 * @return array
+	 */
+	private function get_site_kit_audiences( $audiences ) {
+		$created_audiences = array();
+
+		// Ensure that audiences are available, otherwise return an empty array.
+		if ( empty( $audiences ) || ! is_array( $audiences ) ) {
+			return $created_audiences;
+		}
+
+		$created_audiences = array_filter( $audiences, fn( $audience ) => ! empty( $audience['audienceType'] ) && ( 'SITE_KIT_AUDIENCE' === $audience['audienceType'] ) );
+
+		if ( empty( $created_audiences ) ) {
+			return array();
+		}
+
+		return wp_list_pluck( $created_audiences, 'displayName' );
```

Comment:
I'd suggest a small refactor here, note that `wp_list_pluck()` will return an empty array when passed one so there's no need for the explicit check:

```suggestion
		// Ensure that audiences are available, otherwise return an empty array.
		if ( empty( $audiences ) || ! is_array( $audiences ) ) {
			return array();
		}

		$site_kit_audiences = array_filter( $audiences, fn( $audience ) => ! empty( $audience['audienceType'] ) && ( 'SITE_KIT_AUDIENCE' === $audience['audienceType'] ) );

		return wp_list_pluck( $site_kit_audiences, 'displayName' );
```